### PR TITLE
Day-1: Create a simple AWS EC2 Instance with README.md and .gitignore file

### PR DESCRIPTION
**Pull request by sagary2j**
Summary:

- README.md file created
- .gitignore file created
- terraform code to deploy simple ec2 instance with default vpc.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.terraform
+.terraform.*
+terraform.*
+.git
```
Comment (geoffrey1330) [inline .gitignore]: could you please, add newline at the end of file.

File: Day-1/main.tf [hunk 2]
```diff
@@ -0,0 +1,8 @@
+resource "aws_instance" "app_server" {
+  ami           = "ami-06b09bfacae1453cb"
+  instance_type = "t2.micro"
+
+  tags = {
+    Name = "Task to deploy first simple AWS resource"
+  }
+}
```
Comment (geoffrey1330) [inline Day-1/main.tf]: could you please, add newline at the end of file.

File: Day-1/provider.tf [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    }
+  }
+
+  required_version = ">= 1.2.0"
+}
+
+provider "aws" {
+  region  = "us-east-1"
+  profile = "default"
+}
```
Comment (geoffrey1330) [inline Day-1/provider.tf]: could you please, add newline at the end of file.

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (sagary2j) [reply .gitignore]: Added!

File: Day-1/provider.tf (hunk 3 above)
Comment (sagary2j) [reply Day-1/provider.tf]: Added!

File: Day-1/main.tf (hunk 2 above)
Comment (sagary2j) [reply Day-1/main.tf]: Added!

**Review by geoffrey1330 — APPROVED** [review]

**Pull request merged**